app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -368,6 +369,12 @@ def alert_default_repeat_time
     end
   end
 
+  def apply_default_repeat_time?
+    (@edit[:new][:expression][:eval_method] && @edit[:new][:expression][:eval_method] == "hourly_performance") ||
```

Comment:
can you write it to using early return instead, e.g. something like:

```ruby
def method
  event = @edit[:new]
  return true if event[:exp_event] == '_hourly_timer_'
  return false if event[:eval_method].nil?
  event[:eval_method] == /hourly_performance|dwh_generic/
end
```

Also - missing tests.